npc/airports/airships.txt

```diff
@@ -1086,79 +1089,79 @@ airplane_01,32,61,4	script	Nils#ein	1_M_03,1,1,{
 			"text as quickly as you\r"
 			"can without making any\r"
 			"mistakes! Let's start~!");
-		setarray .@line1_1$[0],	"^3cbcbccallipygian salacius lascivious^000000",
+		setarray .@line1_1$[0], 	"^3cbcbccallipygian salacius lascivious^000000",
```

Comment:
Please remove the tab here (generally, tabs should be used only for indentation and not in the middle of the lines, since they cause things to line up poorly depending on each developer's editor settings)